Panic error when running Portainer with default data folder on Windows

As reported in #459 

I am getting panic errors when running inside a container though:

``` powershell
 docker run -it --rm -p 9000:9000 --name portainer portainer/portainer:pr459

panic: runtime error: slice bounds out of range

goroutine 1 [running]:
panic(0x7c92e0, 0xc042012020)
        /usr/local/go/src/runtime/panic.go:500 +0x1af
syscall.Readlink(0x83a1da, 0x7, 0xc04213ee80, 0x80, 0x80, 0x0, 0x0, 0x0)
        /usr/local/go/src/syscall/syscall_windows.go:1031 +0x3a9
os.Readlink(0x83a1da, 0x7, 0xa1bac0, 0xc04202da40, 0x0, 0x0)
        /usr/local/go/src/os/file_posix.go:21 +0x94
os.Stat(0x83a1da, 0x7, 0xc04218dd38, 0x41048f, 0x20, 0x7e3560)
        /usr/local/go/src/os/stat_windows.go:72 +0x9e
github.com/portainer/portainer/file.createDirectoryIfNotExist(0x83a1da, 0x7, 0x1ed, 0xc04220b008, 0x7)
        /go/src/github.com/portainer/portainer/file/file.go:115 +0x40
github.com/portainer/portainer/file.NewService(0x83a1da, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/portainer/portainer/file/file.go:37 +0x121
main.main()
        /src/cmd/portainer/main.go:32 +0x177
```

But running with a specific data directory worked.  Odd.
`docker run -it --rm -p 9000:9000 --name portainer portainer/portainer:pr459 --data=c:\somewheredata`

--- a/api/cli/defaults_windows.go
+++ b/api/cli/defaults_windows.go
@@ -2,11 +2,11 @@ package cli
 
 const (
 	defaultBindAddress     = ":9000"
-	defaultDataDirectory   = "C:\\data"
+	defaultDataDirectory   = "C:\\ProgramData\\Portainer"
 	defaultAssetsDirectory = "."
 	defaultTemplatesURL    = "https://raw.githubusercontent.com/portainer/templates/master/templates.json"
 	defaultTLSVerify       = "false"
-	defaultTLSCACertPath   = "C:\\certs\\ca.pem"
-	defaultTLSCertPath     = "C:\\certs\\cert.pem"
-	defaultTLSKeyPath      = "C:\\certs\\key.pem"
+	defaultTLSCACertPath   = "C:\\ProgramData\\Portainer\\certs\\ca.pem"
+	defaultTLSCertPath     = "C:\\ProgramData\\Portainer\\certs\\cert.pem"
+	defaultTLSKeyPath      = "C:\\ProgramData\\Portainer\\certs\\key.pem"
 )

